packages/next/next-server/lib/constants.ts

```diff
@@ -8,6 +8,8 @@ export const EXPORT_MARKER = 'export-marker.json'
 export const EXPORT_DETAIL = 'export-detail.json'
 export const PRERENDER_MANIFEST = 'prerender-manifest.json'
 export const ROUTES_MANIFEST = 'routes-manifest.json'
+export const CLIENT_ROUTES_MANIFEST = '_routesManifest.js'
```

Comment:
Let's not create yet another file that has to be loaded. This should be a part of the buildManifest file that is loaded on the client-side.

Reply:
Updated to include these in the existing client `buildManifest`